src/lubed/cli.py

```diff
@@ -31,10 +31,13 @@ def cli():
 )
 def init(last_timestamp_file, force):
     """Initialize the last-timestamp-file with the current time."""
-    with open(last_timestamp_file, "r") as f:
-        if f.read() and not force:
-            console.print(f"Use --force to override {last_timestamp_file}.")
-            exit(3)
+    try:
+        with open(last_timestamp_file, "r") as f:
+            if f.read() and not force:
+                console.print(f"Use --force to override {last_timestamp_file}.")
+                exit(3)
+    except FileNotFoundError:
+        ...
```

Comment:
I don't know how I missed that :+1: 

BTW, this could also be done with https://docs.python.org/3/library/contextlib.html#contextlib.suppress. It's functionally the same, just wanted to make you aware it exists.

Reply:
Nice, implemented!